rollup/types.go

```diff
@@ -278,8 +278,9 @@ func (a *appendSequencerBatchCallData) Decode(r io.ReaderAt) error {
 type CTCTransactionType uint8
 
 const (
-	CTCTransactionTypeEIP155 CTCTransactionType = 0
-	CTCTransactionTypeEOA    CTCTransactionType = 1
+	CTCTransactionTypeEIP155  CTCTransactionType = 0
+	CTCTransactionTypeEthSign CTCTransactionType = 1
+	CTCTransactionTypeEOA     CTCTransactionType = 2
```

Comment:
https://github.com/ethereum-optimism/contracts-v2/blob/c1a9e0fa670e3867e032584568edb60954b04820/contracts/optimistic-ethereum/OVM/precompiles/OVM_SequencerEntrypoint.sol#L107-L111 
To match with the current sequencer entrypoint (decompressor), should be
```suggestion
	CTCTransactionTypeEIP155  CTCTransactionType = 0
	CTCTransactionTypeEthSign CTCTransactionType = 2
```

Reply:
We can also remove all CreateEOA logic in this file

Reply:
Added an issue for this here: https://github.com/ethereum-optimism/go-ethereum/issues/112